nixos/modules/services/networking/networkmanager.nix

```diff
@@ -25,7 +25,7 @@ let
   enableIwd = cfg.wifi.backend == "iwd";
 
   # /var/lib/misc is for dnsmasq.leases.
-  stateDirs = "/var/lib/NetworkManager /var/lib/dhclient /var/lib/misc";
+  stateDirs = "/var/lib/NetworkManager /var/lib/dhclient /var/lib/misc /var/lib/NetworkManager-fortisslvpn";
```

Comment:
I know it's out of scope for this change, but can we use tmpfiles?

Reply:
Maybe in a later PR. I wanted this to be backportable, and thus the most simple possible fix.